src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/DisplayAttributeInputs.tsx

```diff
@@ -148,6 +148,20 @@ const DisplayAttributeInputs = ({
             label="Integer"
           />
         );
+      case "Number":
```

Comment:
Could we combine Integer and Number. I don't see any difference expect the label which is attributeType itself.

Reply:
Yeah, that's the only difference, wasn't sure if the label was significant.

Reply:
My Suggestion to keep cases low
```
      case "Number":
      case "Integer":
        return (
          <IntegerInput
            intValue={null}
            canEdit={true}
            handleChange={(val) => {
              setAttributeValue(parseInt(val));
              if (onChangeForComponentType) {
                onChangeForComponentType(val);
              }
            }}
            label={attributeType}
          />
        );
```

Reply:
Oh duh. Thank you!